pkg/downloader/downloader_test.go

```diff
@@ -19,42 +19,43 @@ func getTestDir(subDir string) string {
 
 	return testDir
 }
-
 func TestOciDownloader(t *testing.T) {
-	path_oci := getTestDir("test_oci")
-	if err := os.MkdirAll(path_oci, os.ModePerm); err != nil {
+	ociTestDir := getTestDir("oci_test_dir")
+	if err := os.MkdirAll(ociTestDir, os.ModePerm); err != nil {
 		t.Fatal(err)
 	}
 
 	defer func() {
-		_ = os.RemoveAll(path_oci)
+		_ = os.RemoveAll(ociTestDir)
 	}()
 
-	ociDownloader := OciDownloader{
+	downloader := OciDownloader{
 		Platform: "linux/amd64",
 	}
 
-	err := ociDownloader.Download(*NewDownloadOptions(
+	options := NewDownloadOptions(
 		WithSource(pkg.Source{
 			Oci: &pkg.Oci{
 				Reg:  "ghcr.io",
 				Repo: "zong-zhe/helloworld",
 				Tag:  "0.0.3",
 			},
 		}),
-		WithLocalPath(path_oci),
-	))
+		WithLocalPath(ociTestDir),
+	)
+
+	err := downloader.Download(*options)
 
 	assert.Equal(t, err, nil)
-	assert.Equal(t, true, utils.DirExists(filepath.Join(path_oci, "artifact.tgz")))
+	assert.Equal(t, true, utils.DirExists(filepath.Join(ociTestDir, "artifact.tgz")))
 
-	path_git := getTestDir("test_git")
-	if err := os.MkdirAll(path_oci, os.ModePerm); err != nil {
+	gitTestDir := getTestDir("git_test_dir")
+	if err := os.MkdirAll(gitTestDir, os.ModePerm); err != nil {
 		t.Fatal(err)
 	}
 
 	defer func() {
-		_ = os.RemoveAll(path_git)
+		_ = os.RemoveAll(gitTestDir)
 	}()
 
 	gitDownloader := GitDownloader{}
```

Comment:
Separate this test case for `GitDownloader` and do not write it together with `TestOciDownloader` for `OciDownloader`

Reply:
@zong-zhe , done the same, don't know why there's a timeout occuring for my TestGitDownloader() function, can you please have a look